core/designsystem/src/main/java/com/ku_stacks/ku_ring/designsystem/components/comment/Comment.kt

```diff
@@ -153,12 +155,14 @@ private fun CommentHeader(
             )
         )
         Spacer(modifier = Modifier.weight(1f))
-        Icon(
-            imageVector = ImageVector.vectorResource(R.drawable.ic_message_circle_v2),
-            contentDescription = stringResource(R.string.comment_reply_icon),
-            modifier = Modifier.clickable { onReplyIconClick() },
-            tint = KuringTheme.colors.textBody,
-        )
+        if (!isReply) {
+            Icon(
+                imageVector = ImageVector.vectorResource(R.drawable.ic_message_circle_v2),
+                contentDescription = stringResource(R.string.comment_reply_icon),
+                modifier = Modifier.clickable { onReplyIconClick() },
```

Comment:
람다는 제거하는게 좋을 것 같습니다!

```suggestion
                modifier = Modifier.clickable(onReplyIconClick),
```